### ISSUE TITLE
UX/Accessibility issue - External link textbox has no label at all for screen readers

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1523]
**Describe the bug**
If I want to browse the site with a screen reader (VoiceOver), the External Link textbox has no label tag for the screen reader to read out loud, so I don't know what I have to type in. 

**To Reproduce**
1. Turn on your screen reader (VoiceOver)
2. Go to 'https://www.topcoder-dev.com/settings/account'
3. Click on 'Linked Account'
4. Navigate into the External link textbox. 
5. See error

**Expected behavior**
The External Link textbox should have a label tag for screen readers.

**Screenshots**
<img width="1280" alt="screenshot 2018-09-16 21 53 01" src="https://user-images.githubusercontent.com/35376779/45600436-ecc48780-b9fc-11e8-9947-79f13ea97689.png">

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas
